README.md

```diff
@@ -26,6 +26,18 @@ The cache is located in one of these locations:
 
 Software Bill of Materials (SBOMs) are imported into the cache, manipulated, and then exported.
 
+### Format Agnostic
+
+Generating SBOMs for complex systems usually involves [Multiple SBOM Files](#multiple-sbom-files), and sometimes you have no control over the format of SBOMs provided for sub-components. Internally, `bomctl` parses and stores the SBOMs in a way that is agnostic to any of the formats we support for importing and exporting. This means you can import SBOMs in different formats and easily perform operations on them without having to first convert them and worry about losing information in the process. In addition, this allow for easy conversion of provided SBOMs and easy generation of SBOMs for your system in all the support formats so your users can take what they need.
```

Comment:
- natural casing for link text
- line length (`bomctl` is enforced at 120 or less, I think `markdownlint` defaults to 80 or less)
- recommend rephrasing `you` statements
- couple of typos

```suggestion
Generating SBOMs for complex systems usually involves [multiple SBOM files](#multiple-sbom-files), and sometimes the
format of SBOMs provided for sub-components is outside of a user's control. Internally, `bomctl` parses and stores
SBOMs in a format-agnostic way. `bomctl` can import SBOMs in various formats and easily perform operations on them
without having to first be converted, potentially losing information in the process. In addition, this allows for
easy conversion of provided SBOMs and generation of SBOMs in all supported formats.
```